infrastructure/variables.tf

```diff
@@ -19,3 +19,15 @@ variable "common_tags" {
 variable "appinsights_instrumentation_key" {
   default = ""
 }
+
+variable "postgres_db_component_name" {
```

Comment:
do we need this? the wa-workflow-api does not have it, though

Reply:
as per suggestion from @raghera we want to name this database host and db differently so I saved it as a variable so it's easier to control.